themes/opentermsarchive/layouts/partials/memo.html

```diff
@@ -0,0 +1,36 @@
+{{ $template := default "single" .template }}
+
+<article class="memo{{ with .class }} {{ . }}{{ end }}">
+  <header>
+    {{ if eq $template "list" }}<h4 class="memo__title"><a href="{{ .memo.RelPermalink }}">{{ .memo.Title }}</a></h4>{{ end}}
```

Comment:
Inconsistent whitespace in `{{ end}}`, plus all check failures outlined by GitHub Actions.